optimization/+DoseObjectives/matRad_DoseObjective.m

```diff
@@ -25,7 +25,7 @@
        
     methods (Static)
         function rob = availableRobustness()
-            rob = {'none','STOCH','VWWC','VWWC_INV','COWC','OWC'}; %By default, no robustness is available
+            rob = {'none','STOCH','PROB','VWWC','VWWC_INV','COWC','OWC'}; %By default, no robustness is available
```

Comment:
I left "Prob" out as a default available robustness since the only ones we have some consistent formulation for are the squared deviation objectives, right? I mean you can use a DVH objective on an expected dose, but mathematically I am not sure about the meaning...